README.md

```diff
@@ -11,6 +11,32 @@ part of the array each recursive call considers.
 
 ## Runtime Analysis
 
-Analyse the time complexity of your implementation and give a $\Theta$ bound for
+Analyze the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+Complexity Analysis: 
+
+Looking through my code again deeply I know there are two components to this function. Which are the outer loops and the merge function. 
+
+With the merge function, this could have a loop. We can look at it in a loop logic thought process. A loop inside of a loop is just n * n as the worst-case scenario. 
+
+The outer loop will keep doubling from 1 until it gets to the size of the array which is an exponent-like reaction. 
+
+The merge function just merges the array in order and will move the element if it is greater than the one that it is being compared to. This function will have a runtime of n depending on the length of the array. 
+
+Then the inner loop pretty much creates two arrays and has the value in one of the arrays and compares it to the value in the other array and switches with it if is less than that other value and keeps moving down the array until it can not be switched anymore. This follows the n value again too. This is where I get n * n because of the two loops in the merge function. 
+
+The worst case for this is n^2 because we multiply the loops together. 
+
+With each division of the array until it gets to 1 is where _O_(log n) comes from. 
```

Comment:
Still not clear on this. It sounds like this is completely separate from the loops from your description, but in the code it's not.